docs/Internal Systems/Getting_started_with_Internal_Sytems_Repo.md

```diff
@@ -0,0 +1,65 @@
+# Getting started with Internal Sytems Repo
+This document contains all the information to understand Internal Sytems repostiory and what you need to know to start coding.
+
+
+# Environment Setup
```

Comment:
Prettier will not like that there are multiple h1 (#) headings within this document. I suggest dropping all heading levels by 1 (add another #) throughout the whole file.